### ISSUE TITLE
v702: syntax error not detected in SELECT INTO statement

### ISSUE THREAD

**Issue by fdupret** [abaplint/abaplint#3822]
In v702, this syntax is incorrect:
```
SELECT SINGLE xvore xvorb xecht
  FROM reguv
  INTO ( ev_xvore, ev_xvorb, ev_xecht )
  where laufd = iv_laufd
    and laufi = iv_laufi.
```
`Comma without preceding colon (after SELECT ?).`

The minimal correct syntax is:
```
SELECT SINGLE xvore xvorb xecht
  FROM reguv
  INTO (ev_xvore, ev_xvorb, ev_xecht )
  WHERE laufd = iv_laufd
    AND laufi = iv_laufi.
```
Trailing space is allowed before the closing bracket, but leading space isn't after the opening one.
